R/bootstrap.R

```diff
@@ -772,29 +751,20 @@ tabsetPanel <- function(...,
 #' Create a navigation list panel that provides a list of links on the left
 #' which navigate to a set of tabPanels displayed to the right.
 #'
-#' @param ... [tabPanel()] elements to include in the navlist
-#' @param id If provided, you can use `input$`*`id`* in your
-#'   server logic to determine which of the current navlist items is active. The
-#'   value will correspond to the `value` argument that is passed to
-#'   [tabPanel()].
-#' @param selected The `value` (or, if none was supplied, the `title`)
-#'   of the navigation item that should be selected by default. If `NULL`,
-#'   the first navigation will be selected.
+#' @inheritParams tabsetPanel
+#' @param ... [tabPanel()] elements to include in the navbar.
+#'   Plain strings will be converted to headers.
```

Comment:
This should mention that strings are only allowed in `navlistPanel()` (otherwise, it's an error)

Reply:
It should also probably also mention that, in that case, `"---"` is rendered as a divider

Reply:
Isn't `----` deprecated?